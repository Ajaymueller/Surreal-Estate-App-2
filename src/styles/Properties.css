.Properties {
    margin-left: 160px;
    padding: 12px;
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
}