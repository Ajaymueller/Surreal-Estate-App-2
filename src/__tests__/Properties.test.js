import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Properties from '../Components/Properties';
import { MemoryRouter } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';

const props = { userId: 12345 }

describe("properties", () => {
    it("component renders with correct props", () => {
        const { asFragement } = render( 
        <MemoryRouter><Properties {...props} /></MemoryRouter>)
        expect(asFragement).toMatchSnapshot();
    });
    it("renders properties div", () => {
        const { getByTestId } = render( 
        <MemoryRouter><Properties {...props} /></MemoryRouter>)
        const propertiesDiv = getByTestId("Properties");
        expect(propertiesDiv).toBeInTheDocument();
    });
});