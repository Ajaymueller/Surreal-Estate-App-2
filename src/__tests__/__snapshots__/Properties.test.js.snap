exports[`properties component renders with correct props 1`] = `undefined`;
